serrano/resources/preview.py

```diff
@@ -28,13 +27,41 @@ class PreviewResource(BaseResource, PaginatorResource):
 
     parametizer = PreviewParametizer
 
-    def get(self, request):
+    def get(self, request, **kwargs):
         params = self.get_params(request)
 
-        page = params.get('page')
         limit = params.get('limit')
         tree = params.get('tree')
 
+        page = kwargs.get('page')
+        stop_page = kwargs.get('stop_page')
+
+        offset = None
+
+        # Restrict the preview results to a particular page or page range. If
+        # no page range is supplied, use the default limit.
```

Comment:
I think I must have lost something in the rebase. Sorry for the confusion. What I meant meant to ask here is if we should explicitly set the limit to `None` when no page is supplied as is done [here](https://github.com/chop-dbhi/serrano/blob/master/serrano/resources/exporter.py#L103-L106). So:

```python
if page:
    ...
else:
    limit = None
```

Under the setup that currently exists, limit will always be 20 when not supplied as you noted.